src/app/app.module.ts

```diff
@@ -6,37 +6,23 @@ import { StatusBar } from '@ionic-native/status-bar';
 import { SplashScreen } from '@ionic-native/splash-screen';
 
 import { MyApp } from './app.component';
-import { HomePage } from '../pages/home/home';
-import { DirectoryPage } from '../pages/directory/directory';
-import { LoginPage } from '../pages/login/login';
 import { ComponentsModule } from '../components/components.module';
 import { PipesModule } from '../pipes/pipes.module';
-import { DirectoryDetailPage } from "../pages/directory-detail/directory-detail";
 import { ApiServiceProvider } from '../providers/api.service/api.service';
 import { ProfilesApi, ProfilesInjectionToken } from '../app/app-config';
 
 @NgModule({
   declarations: [
-    MyApp,
-    HomePage,
-    DirectoryPage,
-    LoginPage,
-    DirectoryDetailPage
```

Comment:
Now all of these pages/components only have to be imported once in their own modules, so they only get loaded by the browser when absolutely needed.